scalableminds/webknossos
# Mitigate rare rendering bug

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- import [this DS](https://scm.slack.com/archives/C5AKLAV0B/p1686307231869279) via zarr streaming
- zoom to mag 1,8,8 and verify that no rendering bug is visible anymore

------
(Please delete unneeded items, merge only when none are left open)
